ext/vulnsrc/rhel/rhel.go

```diff
@@ -129,8 +131,18 @@ func (u *updater) Update(datastore database.Datastore) (resp vulnsrc.UpdateRespo
 			continue
 		}
 
+		var reader io.Reader
+		switch r.Header.Get("Content-Type") {
+		case "application/gzip":
+			reader, _ = gzip.NewReader(r.Body)
```

Comment:
This error should be checked.

Reply:
@hdonnay Thanks! Both issues have been addressed.